From 4ed44a48151706e57af32025332615320f7d18ff Mon Sep 17 00:00:00 2001
From: Stef Walter <stef@memberwebs.com>
Date: Tue, 31 Aug 2010 22:27:04 +0000
Subject: [PATCH 1/2] Revert "[pam] Fix build on Snow Leopard."

This reverts commit dc9f71c5b77ea481e71cc7c250de4a967991b16a.

Reverted due to: https://bugzilla.gnome.org/show_bug.cgi?id=628384
---
 configure.in         |    2 +-
 pam/gkr-pam-module.c |    7 -------
 2 files changed, 1 insertions(+), 8 deletions(-)

diff --git a/configure.in b/configure.in
index 8d0e430..447834c 100644
--- a/configure.in
+++ b/configure.in
@@ -251,7 +251,7 @@ AC_ARG_ENABLE(pam,
 
 pam_status="no"
 if test "$enable_pam" != "no"; then
-	AC_CHECK_HEADERS(security/pam_modules.h pam/pam_modules.h, have_pam=yes, have_pam=no)
+	AC_CHECK_HEADER(security/pam_modules.h, have_pam=yes, have_pam=no)
 	if test "$enable_pam" = "yes" -a "$have_pam" = "no"; then
 		AC_MSG_ERROR(The PAM headers are missing)
 	elif test "$have_pam" = "yes"; then
diff --git a/pam/gkr-pam-module.c b/pam/gkr-pam-module.c
index 5b8e4c5..d6fc17d 100644
--- a/pam/gkr-pam-module.c
+++ b/pam/gkr-pam-module.c
@@ -33,15 +33,8 @@
 
 #include "daemon/control/gkd-control-codes.h"
 
-#ifdef HAVE_SECURITY_PAM_MODULES_H
 #include <security/pam_appl.h>
 #include <security/pam_modules.h>
-#else
-#ifdef HAVE_PAM_PAM_MODULES_H
-#include <pam/pam_appl.h>
-#include <pam/pam_modules.h>
-#endif
-#endif
 
 #include <sys/types.h>
 #include <sys/stat.h>
-- 
1.7.2.2

